script.js

```diff
@@ -1,4 +1,68 @@
 var main = function (input) {
   var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  var words = computerGenResult();
+  if(input == 'scissors' && words == 2){
+    console.log('scissors vs paper')
+    console.log('scissors win')
+    return 'you won!!';
```

Comment:
Notice that you can combine the cases of scissors vs paper, paper vs stone, and stone vs scissors into a single if conditional so that you don't repeat the code `return 'you won!!'`! Please do not repeat code. It is generally bad and you will lose your job because of it.